setup.py

```diff
@@ -74,5 +74,5 @@ def read_file(path_segments: Sequence[str]) -> str:
         setup_requires=["setuptools_scm"],
         install_requires=INSTALL_REQUIRES,
         extras_require=EXTRAS_REQUIRE,
-        long_description=read_file(("README.rst",)),
+        long_description=read_file(("README.md",)),
```

Comment:
I think you would want to include what the content type of this file is. (I think it is `long_description_content_type`?)